docs/catalyst_101.md

```diff
@@ -136,6 +136,159 @@ Once the existing members have voted to offer you membership, it’s up to you t
 
 ## Project Management
 
+We have a quarterly planning process, where we set high level goals for the quarter.
+
+When it comes time to execute on those goals, we plan our work in two-week sprints, described here.
+
+We don't have any planning process in between these two levels of granularity, yet. The goal of that would be to break down our quarterly goals into smaller, logical steps and work towards them consistently over the quarter.
+
+### Overall goals:
+
+- It should be easy to decide what to work on next.
+- We should feel some sense of progress towards our high level goals in our day-to-day.
+- We should be able to identify dependencies and bottlenecks.
+- Each team should be able to communicate its goals on a two-week timeline to the other teams.
+- We should be able to stick to the process, because the overhead feels worth the effort.
+
+Inframundo is currently leading a phase of experimentation with sprint process; as the process evolves, other teams such as RMI and Bizdev may incorporate this process into the way they work.
+
+### What we do now:
+
+We have three synchronous meetings:
+
+Sprint planning (Monday, week 1):
+
+- for each high level initiative, talk about each work item that we want to pull into this sprint
+  - what is it?
+  - should we do it?
+  - tag it with the sprint
+- any team discussions that people want to bring up
+
+Mid sprint checkin (Monday, week 2):
+
+- any team discussions that people want to bring up
+- for each item, make sure it's in the right place
+
+Sprint review (Thursday, week 2):
+
+- for each item tagged with the sprint, talk about:
+  - what it was
+  - what its status is
+- additional ad-hoc discussion topics
+
+Outside of meetings:
+
+- issues and PRs get ingested to the ZenHub board automatically
+- when issue status changes, move the issue in the ZenHub board (maybe - if not, we get to it in the meetings)
+
+### How we'll do it in GH Projects:
+
+Pretty much the same, but with the Catalyst Backlog project instead of ZenHub.
+
+Sprint planning (Monday, week 1):
+
+- for in-flight (in progress, or in review) items in the old sprint:
+  - discuss status and whether we should roll into the new sprint; tag with new sprint if necessary.
+- for items in backlog:
+  - discuss whether we should pull into the new sprint; tag with new sprint if necessary.
+- ad-hoc discussion topics
+
+Mid sprint checkin (Monday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Sprint review (Thursday, week 2):
+
+- for each item in the sprint:
```

Comment:
Similar to previous comment, but I like the idea of having some discussion in each of these meetings about how progress relates to higher level goals

Reply:
Yeah, I think this is a good idea! Our meetings might start to get kind of long if we're adding backlog tending as an actual activity that we do - I kind of like separating backlog tending & sprint retro from each other, but we can see how shoving everything into sprint review feels for now.

Reply:
I guess "have high level goals for the sprint" would be a new experiment that captures both of these.